backend/src/main/java/de/janka/capstonedrazen/repo/UserRepository.java

```diff
@@ -0,0 +1,16 @@
+package de.janka.capstonedrazen.repo;
+
+import de.janka.capstonedrazen.model.UserEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+
+import java.util.Optional;
+
+
+public interface UserRepository extends JpaRepository<UserEntity, Long> {
+
+    Optional<UserEntity> findByUserName(String userName);
+
+    Optional<UserEntity> findByUserNameContains(String name);
```

Comment:
warum die zweite Methode? Da suchst du ob innerhalb einer Datenbank ein User mit gewißen Strings drin ist, bspw ob im in der DatenBank ein user gibt der Alex beinhält (bei Alexander in der DB gäbe es den User zurück). Zumindest glaube ich dass es so funktioniert